systemtest/src/main/java/io/strimzi/systemtest/Environment.java

```diff
@@ -287,13 +295,29 @@ public static boolean useLatestReleasedBridge() {
         return Environment.BRIDGE_IMAGE.equals(Environment.BRIDGE_IMAGE_DEFAULT);
     }
 
+    public static boolean isIpv4Family() {
+        return IP_FAMILY.contains(IP_FAMILY_DEFAULT);
+    }
+
     private static String getOrDefault(String varName, String defaultValue) {
         return getOrDefault(varName, String::toString, defaultValue);
     }
 
     public static String getImageOutputRegistry() {
         if (KubeClusterResource.getInstance().isOpenShift()) {
             return "image-registry.openshift-image-registry.svc:5000";
+        } else if (KubeClusterResource.getInstance().isKind()) {
+            // we will need a hostname of machine
+            String hostname = "";
+            try {
+                if (Environment.isIpv4Family()) {
+                    hostname = InetAddress.getLocalHost().getHostAddress() + ":5001";
+                }
+            } catch (UnknownHostException e) {
+                throw new RuntimeException(e);
+            }
+            LOGGER.info("Using hostname:{}", hostname);
```

Comment:
Is that ome left over debug message? If not, should i tbe at least properly formatted and have some meaning?

```suggestion
            LOGGER.info("Using container registry hostname: {}", hostname);
```

Reply:
Updated :) thanks